task24/js/main.js

```diff
@@ -0,0 +1,38 @@
+import "../css/style.css";
+
+const modalWrapper = document.getElementById("js-modalContents");
+const modal = document.getElementById("js-modal");
```

Comment:
Q.
Do you use it?

Reply:
I forgot about it. Thanks for pointing that out.
[22ae208](https://github.com/seeeen044/JavaScript_handson/pull/26/commits/22ae208b1021a507ebcac0d073754304b3540df4)